system-tests/package.json

```diff
@@ -12,14 +12,13 @@
     "lint:fix:eslint": "eslint . --fix",
     "lint:config": "prettier --check '**/*.@(json|yaml)'",
     "lint:config:fix": "prettier --write '**/*.@(json|yaml)'",
-    "test": "hardhat test",
-    "postinstall": "npm rebuild"
+    "test": "hardhat test"
   },
   "files": [
     "test/**/*"
   ],
   "dependencies": {
-    "@keep-network/tbtc-v2.ts": "development",
+    "@keep-network/tbtc-v2.ts": "2.0.0-dev.0",
```

Comment:
@lukasz-zimnoch 
Should we bring back the `development` tag and simply keep the changes in `yarn.lock`?

Reply:
Good question! I expect we'll stick to `^2.0.0` while working on https://github.com/keep-network/tbtc-v2/issues/716 so system tests can use all compatible `2.x.x` versions. For now, I'd stick to `2.0.0-dev.0` to explicitly mark the used version.